Xamarin.Forms.Platform.iOS/Renderers/ButtonLayoutManager.cs

```diff
@@ -211,6 +217,38 @@ internal void UpdateText()
 			Control.SetAttributedTitle(disabled, UIControlState.Disabled);
 
 			UpdateEdgeInsets();
+			UpdateLineBreakMode();
+		}
+
+		void UpdateLineBreakMode()
+		{
+			var control = Control;
```

Comment:
Could you create a class with extension methods (as you have done with UWP)? So, we would have the same code at https://github.com/xamarin/Xamarin.Forms/blob/ff63ef551d9b2b5736092eb48aaf954f54d63417/Xamarin.Forms.Platform.iOS/Renderers/LabelRenderer.cs#L334 and here.

Reply:
Can you give me a tip? Because on UWP both `Label `and the text in `Button` are `TextBlock`, so was easy create a extension method. But I can't see a way to do that on iOS